packages/mendel-core/trees.js

```diff
@@ -42,7 +42,7 @@ MendelTrees.prototype.findTreeForVariations = function(bundle, variations) {
     return finder.found();
 };
 
-MendelTrees.prototype.findServerVariationMap = function(variations) {
+MendelTrees.prototype.findServerVariationMap = function(variations, bundleList) {
```

Comment:
Yes, I know a new param is needed.

My question was about the param type. Instead of "bundles" array, could it be "bundle" string.

In other words: can the new param be `findServerVariationMap("my_app", ["base"])` or does it need to be `findServerVariationMap(["my_app", "my_other_app"], ["base"])` ?